pkg/data/management/machinedriver_data.go

```diff
@@ -93,6 +96,9 @@ func addMachineDrivers(management *config.ManagementContext) error {
 	if err := addMachineDriver(GoogleDriver, "local://", "", "", nil, false, true, true, management); err != nil {
 		return err
 	}
+	if err := addMachineDriver(HarvesterDriver, "https://harvester-node-driver.s3.amazonaws.com/driver/v0.2.0/docker-machine-driver-harvester-amd64.tar.gz", "", "fdd3428737a15ae28b976476b5b1a4d7d21d9cf9b4c6d9c77cc82e29660f1814", []string{"harvester-node-driver.s3.amazonaws.com"}, false, false, false, management); err != nil {
```

Comment:
Does the harvester UI use this URL `"harvester-node-driver.s3.amazonaws.com"`? If so shouldn't we move this under a rancher/suse owned domain? I saw we were going to embed the UI which is why it's not referenced here.

Reply:
yep, the UI is submitted to the rancher dashboard via https://github.com/rancher/dashboard/issues/3374,
file an EIO ticket to move the release to `releases.rancher.com/harvester-node-driver/` and will update this once the release is ready, thanks